fetch.bs

```diff
@@ -3237,7 +3250,103 @@ through TLS using ALPN. The protocol cannot be spoofed through HTTP requests in
 </ol>
 </div>
 
+<h3 id=https-upgrades>HTTPS upgrading</h3>
+
+<div algorithm>
+
+<p>User agents may optionally upgrade requests with <a>potentially untrustworthy URL</a>s
+to attempt to fetch them over <a>potentially trustworthy URL</a>s. If an upgraded request
+fails with a network error, it is retried over the original URL.
+
+<p>The HTTPS upgrading algorithm consists of Upgrade and Fallback algorithms, with the following
+steps:
+
+<h4 id=https-upgrades-upgrade>Upgrade algorithm</h4>
```

Comment:
```suggestion
<h4 id=https-upgrades-upgrade>HTTPS upgrade algorithm</h4>
```

"Upgrade" already [means something](https://httpwg.org/specs/rfc9110.html#field.upgrade) in HTTP(S)...

Reply:
Done